docs/source/getting_started/getting_started.md

```diff
@@ -0,0 +1,226 @@
+We have two authentication tokens, one to connect to the back office and one to connect to the front office.  
+To authenticate, we need to set the `authenticationToken` header to one of them, depending on what we need to do.
+As our API expects all request bodies to be JSON encoded, we also need to set the `Content-Type` header to `application/json`.
+
+# Ingest
+
+We can use the back office endpoint [`/documents`](https:/docs.xayn.com/back_office.html#operation/createDocuments) to ingest documents.
+
+We will ingest a document that represents this article: [https://xayn.com/blog/the-initial-challenge](https://xayn.com/blog/the-initial-challenge).
+
+```bash
+curl -X POST https://<url>/documents
+    -H "authorizationToken: <back_office_token>"
+    -H "Content-Type: application/json"
+    -d '{
+        "documents": [
+            {
+                "id": "xayn_cd5604c",
+                "snippet": "The voices that are demanding better privacy protection and ownership of our own data are increasingly louder, there's a backlash towards these practices. At Xayn, our mission is to provide personalisation without user data leaving the device, maintaining absolute privacy. We use semantic similarity and centers of interest to understand user preferences and present better matching articles. With our model Xaynia, we offer semantic similarity and search with minimal energy consumption and at a low price, making it highly energy-efficient compared to other transformer models.",
+                summarize: false,
+
+                "properties": {
+                    "title": "The initial challange",
+                    "link": "https://xayn.com/blog/the-initial-challenge",
+                    "image": "https://uploads-ssl.webflow.com/5ef08ebd35ddb63551189655/641320bc6be72c5453f4d98d_Blog%20Posts%20Visuals%20-%2003%20Mar%202023-p-2600.png",
+
+                }
+            }
+        ]
+    }'
+```
+
+The endpoint takes a list of documents to ingest.
+
+Each document has a unique identifier that can be used to refer to it in the system.
+
+The `snippet` field is used to inform the system about the content of the document; it is used as input to Xaynia to generate a mathematical representation of the document that we can use to match similar documents.
+
+For this reason, it is essential that the snippet clearly represents the content of the document. In this case, we took a few representative sentences from the document and used them as a snippet. Since the amount of data that Xaynia can analyse is limited, if it is not possible to provide a concise snippet, we can use the per-document option 'summarise'; when enabled, the system will summarise the content of the document to create a snippet.
+
+The 'properties' field is completely optional. It can contain custom data that can be used for filtering and that the system will return when a document is part of the result of a query.
```

Comment:
:art: probably better to use back-ticks instead of apostrophs for markdown highlighting. similarly for other occurences in the text.
```suggestion
The `properties` field is completely optional. It can contain custom data that can be used for filtering and that the system will return when a document is part of the result of a query.
```